dll/cpl/desk/lang/ro-RO.rc

```diff
@@ -11,10 +11,10 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 FONT 8, "MS Shell Dlg"
 BEGIN
     LTEXT "A theme is a background plus a set of sounds, icons, and other elements to help you personalize your computer with one click.", IDC_STATIC, 5, 5, 235, 30
-    LTEXT "Theme:", IDC_STATIC, 5, 42, 55, 10
+    LTEXT "Temă:", IDC_STATIC, 5, 42, 55, 10
     COMBOBOX IDC_THEMES_COMBOBOX, 5, 52, 160, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON "Save &As...", IDC_THEMES_SAVE_AS, 170, 52, 70, 14
-    PUSHBUTTON "&Delete", IDC_THEMES_DELETE, 170, 70, 70, 14
+    PUSHBUTTON "Salvează &ca...", IDC_THEMES_SAVE_AS, 170, 52, 70, 14
```

Comment:
The term I used when translating „save” was „păstrează”, because in English it includes meanings like „save money”, or „save (your) breath”, whereas in Romanian it doesn't. Also, take note that, becausi it is just an intermediary action, you've already agreed to use long infinitives, so may result as „Păstrează &ca…”.